DataLayer/BatteryData/BatteryData.cpp

```diff
@@ -227,43 +227,43 @@ void BatteryData::setPopulatedCells(const unsigned char& populatedCells)
 
 void BatteryData::set12VInputVoltage(const float& inputVoltage12V)
 {
-    inputVoltage12_ = inputVoltage12V;
+    inputVoltage12_ = (inputVoltage12V * 10);
```

Comment:
I think this is mixed up, if we're going from 0.1V to 1V, for a value of say 1 V, CAN & Serial will report a value of 10, since 10 * 0.1 = 1V. We would then _divide_ by 10, to get the appropriate value.

Reply:
Also, let's use namespace for each these magic numbers.